Bio/Align/__init__.py

```diff
@@ -2871,10 +2871,10 @@ def inverse_indices(self):
             starts = self.coordinates[:, k]
             ends = self.coordinates[:, k + 1]
             for row, start, end, rc in zip(a, starts, ends, rcs):
-                if rc == False and start < end:  # noqa: 712
+                if rc == False and start < end:  # noqa: E712
                     j = i + end - start
                     row[start:end] = range(i, j)
-                elif rc == True and start > end:  # noqa: 712
```

Comment:
@mdehoon Is the usage of ``rc == True`` and ``rc = False`` an optimisation, and thus preferable here over using ``rc`` and ``not rc``, or ``rc is True`` and ``rc is False``?